rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -3422,20 +3422,20 @@ class GiveActionExecutor : public IUnsyncedActionExecutor {
 	}
 };
 
-class DestroyActionExecutor : public IUnsyncedActionExecutor {
+
+class BaseDestroyActionExecutor : public IUnsyncedActionExecutor {
 public:
-	DestroyActionExecutor() : IUnsyncedActionExecutor("Destroy", "Destroys one or multiple units by unit-ID, instantly", true) {
-	}
+	BaseDestroyActionExecutor(const std::string& command, const std::string& description)
+		: IUnsyncedActionExecutor(command, description, true) {}
 
 	bool Execute(const UnsyncedAction& action) const final {
-		if (selectedUnitsHandler.selectedUnits.empty())
+		if (selectedUnitsHandler.selectedUnits.empty()) {
 			return false;
+		}
 
-		// kill selected units
 		std::stringstream ss;
-		ss << GetCommand();
-
-		for (const int unitID: selectedUnitsHandler.selectedUnits) {
+		ss << action.GetCmd();
+		for (const int unitID : selectedUnitsHandler.selectedUnits) {
```

Comment:
The change from `this->GetCommand()` to `action.GetCmd()` is just to make it consistent.

For full details:
- `this->GetCommand()` returns the command as passed to executor, e.g. `Remove`.
- `action.GetCmd()` gets the actual command from the action itself, so e.g. `remove` if you typed `/remove` or `Remove` if you typed `/Remove`.

The executor is picked based on action command in case-insensitive way, so in the code I just made sure to use `action.GetCmd()` when passing the command from unsynced to synced (as it's supposed to be a passthrough), whereas I used `this->GetCommand()` for things related to executor logic (e.g. logging which executor we're in right now).